routers/maze.js

```diff
@@ -1,6 +1,9 @@
 const express = require('express');
 const router = express.Router();
-const { getOperator } = require('../operators/operators');
+const {
+  validateFindRequest,
+  validateCreateRequest,
+} = require('./middleware/validators');
 
 let maze = [];
```

Comment:
It's completely valid case to maintain only the last added instance of maze.
However, the rest of the app must be ready for that. E.g. you are not checking if the maze exists. 
As a result, after the app is started and I first look up values I'd be damned.

Reply:
@zatziky I think the maze always exists, am I wrong? In the beginning it is an empty list and after that it only changes in case valid array was sent in body.

Reply:
If I first try to look up the result will be empty (so no values found), no errors will be thrown and app will continue.

Reply:
@datogogadze Sorry Dato. You are right. It always exists. :-)